packages/component-library/src/components/form/mt-number-field/mt-number-field.vue

```diff
@@ -252,6 +269,28 @@ export default defineComponent({
       },
       immediate: true,
     },
+
+    min() {
+      if (this.currentValue === null || this.currentValue === undefined) {
+        return;
+      }
+
+      if (Number.isNaN(Number(this.currentValue))) {
```

Comment:
Sorry my mistake, my last diff was flawed:
```suggestion
      if (!Number.isNaN(Number(this.currentValue))) {
```